examples/k8s.yaml

```diff
@@ -2,40 +2,40 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: otelsvc-agent-conf
+  name: otel-agent-conf
```

Comment:
nit: in a follow up should we remove the term agent from here and in other samples? And only refer to it in the documentation when describing the deployment model?

Reply:
This example is about actually deploying Collector as an Agent (as DaemonSet). So in this case we really need to name it an agent because that's what it is. :-)